src/fsharp/IlxGen.fs

```diff
@@ -4902,13 +5230,11 @@ and GetIlxClosureFreeVars cenv m (thisVars: ValRef list) eenvouter takenNames ex
 
     let cloFreeTyvars = cloFreeTyvars.FreeTypars |> Zset.elements
 
-    let cloAttribs = []
```

Comment:
I see, it was empty all along. I guess that makes sense - though in the future we might put attributes on closures.. I think C# will too.

Reply:
Right.  We can re-instate when it's no empty